tests/SCF/test_SCF.py

```diff
@@ -17,3 +24,45 @@ def test_calc_initial_density(mol_h2o):
     Duv = SCF.calc_initial_density(mol_h2o)
     assert Duv.sum() == 0.0
     assert Duv.shape == (mol_h2o.nao,mol_h2o.nao)
+
+def test_calc_hcore_matrix():  # checks Hcore matrix
+    Tuv = pickle.load(open("tuv.pkl", "rb"))
+    Vuv = pickle.load(open("vuv.pkl", "rb"))
+    Huv = SCF.calc_hcore_matrix(Tuv, Vuv)
+    assert Huv[0, 0] == -32.57739541261037
+    assert Huv[3, 4] == 0.0
+
+
+def test_calc_fock_matrix(mol_h2o):  # checks fock matrix
+    Huv = SCF.calc_hcore_matrix(Tuv, Vuv)

```

Comment:
Tuv and Vuv should be accessed through the fixture rather than the global variables above.